### PR TITLE
Add GetMessagePos in User32 and remove GetMessagePos from SafeNativeM…

### PR DESCRIPTION
**Pull request by gpetrou**
…ethods

## Proposed changes

- Add GetMessagePos in User32
- Remove GetMessagePos from SafeNativeMethods and replace its usages with the new method.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2207)

### CODE REVIEW

**Comment by zsd4yr** [comment]
@msftbot please merge if @RussKie approves

File: src/Common/src/Interop/User32/Interop.GetMessagePos.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true, CharSet = CharSet.Auto)]
+        public static extern int GetMessagePos();
```
Comment (gpetrou) [inline src/Common/src/Interop/User32/Interop.GetMessagePos.cs]: I see this returns DWORD. Should we change to uint?

Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.GetMessagePos.cs]: Yes

Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.GetMessagePos.cs]: Done.

**Pull request merged**